// @storybook/addon-vitest/dist/_node-chunks/vitest.workspace-NGT3ZVLN.js
import CJS_COMPAT_NODE_URL_62figg9psgv from 'node:url';
import CJS_COMPAT_NODE_PATH_62figg9psgv from 'node:path';
import CJS_COMPAT_NODE_MODULE_62figg9psgv from "node:module";

var __filename = CJS_COMPAT_NODE_URL_62figg9psgv.fileURLToPath(import.meta.url);
var __dirname = CJS_COMPAT_NODE_PATH_62figg9psgv.dirname(__filename);
var require = CJS_COMPAT_NODE_MODULE_62figg9psgv.createRequire(import.meta.url);

// ------------------------------------------------------------
// end of CJS compatibility banner, injected by Storybook's esbuild configuration
// ------------------------------------------------------------
import "./chunk-M5SV5NPD.js";

// raw:../templates/vitest.workspace.template
var vitest_workspace_default = `import path from 'node:path';
import { fileURLToPath } from 'node:url';

import { defineWorkspace } from 'vitest/config';

import { storybookTest } from '@storybook/addon-vitest/vitest-plugin';

const dirname =
  typeof __dirname !== 'undefined' ? __dirname : path.dirname(fileURLToPath(import.meta.url));

// More info at: https://storybook.js.org/docs/next/writing-tests/integrations/vitest-addon
export default defineWorkspace([
  'ROOT_CONFIG',
  {
    extends: 'EXTENDS_WORKSPACE',
    plugins: [
      // The plugin will run tests for the stories defined in your Storybook config
      // See options at: https://storybook.js.org/docs/next/writing-tests/integrations/vitest-addon#storybooktest
      storybookTest({ configDir: path.join(dirname, 'CONFIG_DIR') }),
    ],
    test: {
      name: 'storybook',
      browser: {
        enabled: true,
        headless: true,
        provider: 'playwright',
        instances: [{ browser: 'chromium' }],
      },
    },
  },
]);
`;
export {
  vitest_workspace_default as default
};
